src/enable_variable_ops/ngraph_rewrite_pass.cc

```diff
@@ -282,7 +282,7 @@ class NGraphEncapsulationPass : public NGraphRewritePass {
     // 4. Encapsulate clusters then, if requested, dump the graphs.
     FunctionDefLibrary* fdeflib_new = new FunctionDefLibrary();
     TF_RETURN_IF_ERROR(
-        EncapsulateClusters(options.graph->get(), idx, fdeflib_new));
+        EncapsulateClusters(options.graph->get(), idx, fdeflib_new, nullptr));
```

Comment:
Passing in `nullptr`: Right now we are not using the information we collect (Only unit test is using this info)